servatrice/servatrice.ini.example

```diff
@@ -142,7 +142,37 @@ disallowedregexp=""
 
 ; Set this number to the maximum number of accounts any one user can use to create new accounts
 ; using the same email address. 0 = Unlimited number of accounts (default).
-;maxaccountsperemail=0 
+;maxaccountsperemail=0
+
+[forgotpassword]
+
+; Servatrice can process forgot password requests allowing users to reset their account
+; passwords in the event they forget it.  Should this feature be enabld? Default: false.
+; enable=false
+
+; Forgot password request should not be allowed to stay valid forever.  This settings
+; informs servatrice how long a players forgot password reset token is valid for (in minutes).
+; Default: 60
+; tokenlife=60
+
+; Servatrice can challenge users that are making forgot password requests to answer
+; questions in regards to their account to help validate they are the true owner of the account.
+; Should this feature be enabled? Default: false
+; enablechallenge=false
+
+; Sender email address: the "from" email address for forgot password emails
+email=root@localhost
```

Comment:
I can, given this is different logical functionality I didn't want to re-use parts of something else that appeared to be separated in its own since.  If you want something like a single "from" address should the "from" ini configuration setting be moved out of the registration into the smtp section?